replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/mcard/ReplicationHistoryAttributes.java

```diff
@@ -38,6 +38,22 @@
             false /* tokenized */,
             false /* multivalued */,
             BasicTypes.DATE_TYPE));
+    descriptors.add(
```

Comment:
✏️ Class documentation and copyright notice. Maybe double check all files for that.

Reply:
Looks like this class does have the copyright header. Will add class documentation.